src/components/Feed/Post.tsx

```diff
@@ -35,6 +35,7 @@ const useStyles = makeStyles((theme) => ({
     backgroundRepeat: 'no-repeat',
     backgroundSize: 'contain',
     backgroundPosition: 'right', 
+    width: '56.25rem',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Benedict-Scrumberbatch/peer-recognition/pullRequest?prid=7204339)
```suggestion
    width: "56.25rem",
```